### ISSUE TITLE
add confusion_matrix and CrossEntropyLoss

### ISSUE THREAD

**Issue by marouaneamz** [open-mmlab/mmclassification#1038]
I can't find in the classification documentation how you can add matrixconfusion and differents metric at the end for example : 
```
data = dict(
    samples_per_gpu=14,
    workers_per_gpu=16,
    train=dict(
        type=dataset_type,
        ann_file = './storage/20220906-19classes-train.json',
        img_cache_prefix='./tmp/cache/20220906-19classes',
        classes = classes,
        skip_download = True,
        pipeline=train_pipeline),
    val=dict(
        type=dataset_type,
        img_cache_prefix ='./tmp/cache/20220906-19classes',
        ann_file = './storage/20220906-19classes-val.json',
        classes = classes,
        skip_download = True,
        pipeline=test_pipeline),
    test=dict(
        # replace `data/val` with `data/test` for standard test
        type=dataset_type,
        img_cache_prefix ='./tmp/cache/20220906-19classes',
        ann_file = './storage/20220906-19classes-test.json',
        classes = classes,
        skip_download = True,
        pipeline=test_pipeline))
evaluation = dict(interval=1, metric='accuracy')
```
I would like for the validation to display cross Entropy accuracy and matrix confusion. how can I modify the evaluation

**Comment by Ezra-Yu** [comment]
> I would like for the validation to display cross Entropy accuracy and matrix confusion

1. display cross Entropy accuracy, Is it loss and ACC for val dataset

modify `workflow = [('train', 1)]` to `workflow = [('train', 1), (val, 1)]`

2. matrix confusion

you can add a metric of `matrix_confusion` in [base_dataset](https://github.com/open-mmlab/mmclassification/blob/8d1bc557ab25ad75c17091711629d684cbdba5b8/mmcls/datasets/base_dataset.py#L126) then 
```
evaluation = dict(interval=1, metric=['accuracy', 'matrix_confusion'])
```

**Comment by marouaneamz** [comment]
> > I would like for the validation to display cross Entropy accuracy and matrix confusion
> 
> 1. display cross Entropy accuracy, Is it loss and ACC for val dataset
> 
> modify `workflow = [('train', 1)]` to `workflow = [('train', 1), (val, 1)]`
> 
> 2. matrix confusion
> 
> you can add a metric of `matrix_confusion` in [base_dataset](https://github.com/open-mmlab/mmclassification/blob/8d1bc557ab25ad75c17091711629d684cbdba5b8/mmcls/datasets/base_dataset.py#L126) then
> 
> ```
> evaluation = dict(interval=1, metric=['accuracy', 'matrix_confusion'])
> ```

i do this and i have matrix_confusion not supported

**Comment by tonysy** [comment]
This issue will be closed as it is inactive, feel free to re-open it if necessary.
